test/trans.render.spec.jsx

```diff
@@ -648,6 +648,30 @@ describe('trans should work with selfclosing elements in components', () => {
   });
 });
 
+describe('trans should work with self closing elements with react components', () => {
```

Comment:
can you add another test with your examples? <Link> and <a>

Reply:
what other test should i add?

Reply:
added more tests